.changeset/famous-feet-kneel.md

```diff
@@ -0,0 +1,8 @@
+---
+'lit': minor
+'lit-element': minor
+'@lit/reactive-element': minor
+'@lit/ts-transformers': minor
+---
+
+Add `queryAssignedElements` decorator for a declarative API that calls `HTMLSlotElement.assignedElements()` on a specified slot. `selector` option allows filtering returned elements with a CSS selector.
```

Comment:
Update this since the `selector` option is different now?

Reply:
Thanks for the comment! Would you be able to elaborate? Are you referring to the removal of the Node filter?

In `queryAssignedNodes` the selector changes the return type from `Node[]` to `Element[]`, whilst in this decorator the selector shouldn't change the return type. A docs update to `queryAssignedNodes` is tracked in https://github.com/lit/lit/issues/2171.

Reply:
Oh sorry, I misread this -- I was thinking this was written before you added the `options` parameter, but when I read it again it's fine (I think I read `selector` option as `selector` parameter).